dfm_sentence_trf/evaluation/task_evaluator.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from typing import Dict
+
+from sentence_transformers.evaluation import SentenceEvaluator
+
+from dfm_sentence_trf.tasks.task import Task
+
+logger = logging.getLogger(__name__)
+
+
+class TaskListEvaluator(SentenceEvaluator):
+    """Evaluator that evaluates the model on all test sets
+    from all tasks and returns the sum of their scores."""
+
+    def __init__(self, tasks: Dict[str, Task], log_to_wandb: bool = False):
+        self.tasks = tasks
+        self.log_to_wandb = log_to_wandb
```

Comment:
I would just create a logger object which has a method ".log()". Makes it easier to update the code going forward and allow you to treater "logger" and wandb as the same in the function.
```suggestion
    def __init__(self, tasks: Dict[str, Task], logger: Logger):
        self.tasks = tasks
        self.logger = logger 
```